meilisearch-release-changelog/v1.10.0.md

```diff
@@ -0,0 +1,374 @@
+# v1.10.0 release changelogs
+
+<!-- The following line should ONLY be put PRE-release changelogs -->
+> [!WARNING]
+> Since this is a release candidate (RC), we do NOT recommend using it in a production environment. Is something not working as expected? We welcome [bug reports](https://github.com/meilisearch/meilisearch/issues/new/choose) and [feedback about new features](https://github.com/meilisearch/product/discussions).
+
+With Meilisearch v1.10 we keep innovating by introducing the really demanded federated search! You can know apply multi-search requests and get one single list of results 🎉. This version also includes a setting to define your index languages (even if multiple languages are in your documents!), and new experimental features like the `CONTAINS` operator and the ability to update a subset of your dataset by using a simple function.
```

Comment:
Can you keep the innovation thing? It's something none of our competitors have 😇

Reply:
I can, but I'm against it because it's impossible to do it without sounding like we're giving ourselves a self-high-five. I personally dislike that and feel it's out of place in a document like a changelog.

It's your call, though, so here's a version mentioning that.

```suggestion
Meilisearch v1.10 introduces federated search. This innovative feature allows you to receive a single list of results for multi-search requests. v1.10 also includes a setting to manually define which language or languages are present in your documents, and two new new experimental features: the `CONTAINS` filter operator and the ability to update a subset of your dataset with a function.
```